<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Port Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Button_Send">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="Port">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>61</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Baudlabel">
    <property name="geometry">
     <rect>
      <x>381</x>
      <y>101</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Baud rate:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Port">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>61</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Rate">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>101</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Connect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Exit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Send">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Get">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>340</width>
      <height>211</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
